### DISCUSSION TITLE
Screen goes back when reviewing code

### DISCUSSION THREAD

**Discussion by kernelb00t** [community/community#122068] [Mobile]
### Select Topic Area

Bug

### Body

Hey, I was reviewing code on GitHub's mobile app. I was in a PR, reviewing it and adding comments. When scrolling to the end of the diff page, the app goes back to the previous screen. 

https://github.com/community/community/assets/68474172/919a7467-b87c-4f9b-a623-ba9b8f142d7d
You can see it in this screen record. 
Here's the link to the pr: https://github.com/kernelb00t/KillTheRay/pull/1

**Comment by kangarruu** [comment]
@kernelb00t Thanks for reporting that ❤️. Could you please provide your device model & GitHub Mobile version? Also, by any chance do you have 'Don't keep activities' enabled in your device developer options?

**Reply by kernelb00t** [reply]
Sorry, I forgot to add technical details 😅
![Screenshot_2024-05-04-09-26-00-09_320a9a695de7cdce83ed5281148d6f19.jpg](https://github.com/community/community/assets/68474172/a73bdf8f-5fb5-4dd2-898e-c48bdb758fa3)

I don't have 'Don't keep activities' enabled in the Android dev options.

I have an Android 12 phone (Oppo Reno4 Z 5G), stock rom.

**Reply by kangarruu** [reply]
@kernelb00t Thanks for that info, it helped us track down the crash you were seeing and deploy a fix 💟. Upgrading to the latest beta version when it becomes available to you should resolve this issue. Let us know if you're still impacted once you're on the new version!

**Comment by kernelb00t** [comment]
The issue is fixed with 1.162 beta, thanks

**Reply by Natthaphonkanchaisri** [reply]
Natthaphon99
